src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
+        <TargetPath>src\$([System.String]::new('%(FullPath)').Replace('$(ProjectDir)', ''))</TargetPath>
```

Comment:
Per my comment about source not under project dir, I'm switching to use `SourcesRootPath` or `ProjectDir`, and both of those are full paths. (`SourcesRootPath` manually, and `ProjectDir` is `MSBuildThisFileDirectory` which is a full path.) That should mean that no source files are dropped, but agreed, I'll make it fall back to dropping files directly in `src`. I'll make the substring change too, good catch.